app/components/containers/HTTPError.js

```diff
@@ -1,5 +1,11 @@
+/* eslint-disable global-require */
 import React from 'react';
 
-const HTTPError = ({ code }) => (<h1>Error {code}</h1>);
+const HTTPError = ({ code }) => (
+  <div>
+    <h1 className="error-title">Error {code} </h1>
+    <img src={require('../../../static/media/images/empty_box.svg')} className="error-img"/>
```

Comment:
also, you can just use `<img src='/media/images/empty_box.svg' />` since all static files are rendered through `server.js`